firefox-ios/Client/Frontend/Autofill/Address/AddressCellView.swift

```diff
@@ -0,0 +1,84 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import SwiftUI
+import Common
+import Shared
+import Storage
+
+// MARK: - AddressCellView
+
+/// A view representing a cell displaying address information.
+struct AddressCellView: View {
+    // MARK: - Properties
+
+    @State private var textColor: Color = .clear
+    @State private var customLightGray: Color = .clear
+    var address: Address
```

Comment:
This and a couple other variables in the swiftUI views could be made to `private(set)`

Reply:
Addressed